tfx/components/example_validator/executor_test.py

```diff
@@ -29,45 +29,72 @@
 
 class ExecutorTest(tf.test.TestCase):
 
-  def testDo(self):
-    source_data_dir = os.path.join(
+  def setUp(self):
+    super(ExecutorTest, self).setUp()
+    self.source_data_dir = os.path.join(
         os.path.dirname(os.path.dirname(__file__)), 'testdata')
 
-    eval_stats_artifact = standard_artifacts.ExampleStatistics()
-    eval_stats_artifact.uri = os.path.join(source_data_dir, 'statistics_gen')
-    eval_stats_artifact.split_names = artifact_utils.encode_split_names(
+    self.eval_stats_artifact = standard_artifacts.ExampleStatistics()
+    self.eval_stats_artifact.uri = os.path.join(
+        self.source_data_dir, 'statistics_gen')
+    self.eval_stats_artifact.split_names = artifact_utils.encode_split_names(
         ['eval'])
 
-    schema_artifact = standard_artifacts.Schema()
-    schema_artifact.uri = os.path.join(source_data_dir, 'schema_gen')
+    self.schema_artifact = standard_artifacts.Schema()
+    self.schema_artifact.uri = os.path.join(self.source_data_dir, 'schema_gen')
 
-    output_data_dir = os.path.join(
+    self.output_data_dir = os.path.join(
```

Comment:
nit: `output_data_dir` is not used outside this method and can be local

Reply:
Done